app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,45 @@
+class GroupsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_group, only: [:show]
+
+  def index
+    @groups = Group.all
+  end
+
+  def show
+    @payments = @group.payments.order(created_at: :desc)
```

Comment:
That was because if we try to add a link it won't work.
It would be great to add an image but I don't have time and is Christmas for extra features is not mandatory just set the icon as 'none'.